### ISSUE TITLE
Revision of charts

### ISSUE THREAD

**Issue by 132ndHansolo** [132nd-vWing/TRMA#26]
The charts look awesome. Would it be possible to add the revision no unto the charts themselves. I know the name says which version it is, but if the names are overridden there is no way to identify which version of the charts I am looking at.
I may have overlooked it but could see it directly on any of the charts;
ENBO_VAC_132nd_v04
ENBO_VAC_HPMA_132nd_v05
ENBO_GND_Chart_v06
![B (2)](https://github.com/132nd-vWing/TRMA/assets/28644621/1791d2b7-ef01-42be-9e89-d96b3d2adbb8)
![B (3)](https://github.com/132nd-vWing/TRMA/assets/28644621/b9f6dd19-2543-44f5-9b86-7fd898c37c4c)

**Comment by Shadoga** [comment]
Many thanks for the input.
The chart drafts will receive imprinted version numbers .

**Comment by 132ndNeck** [comment]
Some feedback on charts:
The charts are getting there, and removing the unused "white space" around the charts really helped I think. Some additional feedback:

![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/d1ef18bd-4076-4ca7-82cf-37a154a3998d)
I am not sure the purpose and need for these frequencies? We should consider removing them if not of any purpose.


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/e3619e85-5bbb-4a4f-853c-eaa24bef264b)
I am not sure what this is. I tried tuning to it on yesterdays test event (Tried TACAN in the F-16), but got nothing. Also nothing marked in DCS mission editor. When we make the "final version" of the charts, we can consider deleting items not applicable for us in DCS.


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/d93c8e4a-122a-466c-8286-dd6eb7929ae8)
![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/794444dc-9c66-4341-8e00-ff630c8dedef)

As mentioned in #19 , TWR and APP is enough for 132nd operations, I would recommend remove the frequency completely, or if keeping it in, adding an asterix to it, and informing that the frequency is normally not in use, and the event information page will indicate if the ground frequency is manned.


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/4f40dbd0-272a-4dc2-84c8-c393a2ccfc58)
We do not have this, and this item should be removed. We should only include things applicable to the 132nd in DCS, and not keep things in charts for "realism" sake.


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/3b070cc3-3bbe-42e8-aa0e-9db7bc5d53f5)
As mentioned in #17 we have decided to have our controlling freqs on UHF. We should then also use UHF for ATIS


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/4aebc0cd-c609-4b6a-8cf2-5312e3740607)
It is very hard to read these numbers in VR in a way that I could use them. We should increase their size.


![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/f2640297-ba7e-4cbd-9f84-dec099972934)
It was hard to read these numbers for me in VR in a way that I could use them. We should increase their size.

![bilde](https://github.com/132nd-vWing/TRMA/assets/26002570/861a4f00-a9af-44ae-a1e8-13e24707c355)
It was hard to read this text for me in VR in a way that I could use it. We should increase the size.

**Comment by Shadoga** [comment]
Regarding readability of smaller chart items:
This has mostly to do with the current format being quite crude paintings and not vector graphics.
Resolution / readability is worse in game than with the current kneebord FLIP 1.3 charts for ATRM.
We keep it simple as it is for the current stage of procedure design and not put too much effort in it because of the project teams workflow: me being a total dork regarding graphics, not capable of more than most basic MS Paint functions. Handle seems to be more skillful in regard to graphics, for example he cleaned up the original civ. VAC chart of all procedures so that we could start on a clean one with drawing the HPMA procedures. But so far, there is no coordinated chart drawing process installed in the project.
But for finalizing the charts later after design freeze, they will need to be rebuilt again in formatting, layering, vectors, etc., to offer at least the readability and scalability of the current ATRM charts.
The ENBO TMA Project's focus is procedure and chart layout design, it's current members are not selected for graphics skills, especially not me. Once design is completed, we will need to forward the chart creation to a new group composition specializing in that, e.g. those members who did the current FLIP's charts.
So for the time, please live with the current shortcomings regarding readability, especialy in VR. Those issues should be solved through the later formatting process. What helped me in VR for the time was to drag the kneeboard window size to be a little larger than default...
Other readability issues like e.g. the too small shelter / spawnpoint numbers on the ground chart will be mitigated by additionally creating further more detailed charts for the ENBO shelter loops and spawnpoints in the future FLIP update - just like those secondary detailed ground charts existing currently for OMAM.
So instead of increasing font size in current chart drafts, we should maybe start forming a chart graphics project group that is able to draw the nearly finalized GND, VAC and HPMA VAC charts on a formatting level and resolution like we currently have in the FLIP - and once we have them (is it in vector graphics?), we then should judge readability of the smaller details and fonts.
-------
ATIS will be displayed in UHF once we get it to work and committed to a specific UHF frequency.
-------
Additional Frequencies displayed on ground chart:
In the CENOR FLIP design philosophy, the first chart in each chapter for each airbase's chart compendium is the ground chart.
They not only display the ground layout on it but also publish a collection of further general information and notes about the airbase on that page. One of those being a list of all available services and all their possibly used service frequencies - without any further detailed explanation of which is for what.
We already cut this list down by removing the VHF frequencies (except for ATIS, see above).
I left in the real-world TWR backup UHF for the time, in case we decide to have something like this...
I specifically decided to leave in the 2nd APP UHF frequency because I intend to use that for a possible PRECISION / FINAL controller role.
-------
The BO 40X and BD 24X DMEs are the DME components of the ILSs 07 & 25, the question already came up last time:
https://discord.com/channels/268095055724675082/1236715915270422528/1251895733385433110
They are currently not implemented in DCS. We can remove them from the charts at a later stage if they don't get implemented.
In this case they would need to be removed from the instrument approach charts as well.
-------
Regarding the request to delete the ENBO Ground ATC frequency from the charts, I continue to making my case in Issue #19:
https://github.com/132nd-vWing/TRMA/issues/19#issuecomment-2203506148

**Comment by 132ndNeck** [comment]
> Regarding readability of smaller chart items: This has mostly to do with the current format being quite crude paintings and not vector graphics. Resolution / readability is worse in game than with the current kneebord FLIP 1.3 charts for ATRM. We keep it simple as it is for the current stage of procedure design and not put too much effort in it because of the project teams workflow: me being a total dork regarding graphics, not capable of more than most basic MS Paint functions. Handle seems to be more skillful in regard to graphics, for example he cleaned up the original civ. VAC chart of all procedures so that we could start on a clean one with drawing the HPMA procedures. But so far, there is no coordinated chart drawing process installed in the project. But for finalizing the charts later after design freeze, they will need to be rebuilt again in formatting, layering, vectors, etc., to offer at least the readability and scalability of the current ATRM charts. The ENBO TMA Project's focus is procedure and chart layout design, it's current members are not selected for graphics skills, especially not me. Once design is completed, we will need to forward the chart creation to a new group composition specializing in that, e.g. those members who did the current FLIP's charts. So for the time, please live with the current shortcomings regarding readability, especialy in VR. Those issues should be solved through the later formatting process. What helped me in VR for the time was to drag the kneeboard window size to be a little larger than default... Other readability issues like e.g. the too small shelter / spawnpoint numbers on the ground chart will be mitigated by additionally creating further more detailed charts for the ENBO shelter loops and spawnpoints in the future FLIP update - just like those secondary detailed ground charts existing currently for OMAM. So instead of increasing font size in current chart drafts, we should maybe start forming a chart graphics project group that is able to draw the nearly finalized GND, VAC and HPMA VAC charts on a formatting level and resolution like we currently have in the FLIP - and once we have them (is it in vector graphics?), we then should judge readability of the smaller details and fonts.

Makes sense and very understandable. I just wanted to report it in case it was not known (tested in VR). No rush to get a version working 100% in VR, as long as the feedback is noted and we can work on getting better readability in our charts.

> ATIS will be displayed in UHF once we get it to work and committed to a specific UHF frequency.

Great!

> Additional Frequencies displayed on ground chart: In the CENOR FLIP design philosophy, the first chart in each chapter for each airbase's chart compendium is the ground chart. They not only display the ground layout on it but also publish a collection of further general information and notes about the airbase on that page. One of those being a list of all available services and all their possibly used service frequencies - without any further detailed explanation of which is for what. We already cut this list down by removing the VHF frequencies (except for ATIS, see above). I left in the real-world TWR backup UHF for the time, in case we decide to have something like this... I specifically decided to leave in the 2nd APP UHF frequency because I intend to use that for a possible PRECISION / FINAL controller role.

That makes sense, and no issue with keeping additonal freqs on the chart as long they are explained and annotated for the use. As they stand now, it was hard to know what their purpose were.


> The BO 40X and BD 24X DMEs are the DME components of the ILSs 07 & 25, the question already came up last time: https://discord.com/channels/268095055724675082/1236715915270422528/1251895733385433110 They are currently not implemented in DCS. We can remove them from the charts at a later stage if they don't get implemented. In this case they would need to be removed from the instrument approach charts as well.

Ah, copy. No need to delete it then if it is something we think will come?

**Comment by Shadoga** [comment]
I don't think they will come, co-located ILS-DMEs are no thing in DCS because as far as I'm aware there's no airframe in DCS yet that can use and display them through the ILS receiver.
But it would be nice to have the option to set them separately in the TACAN receiver.
So maybe ORBX / ED have a pleasant surprise one day in that regard. But I doubt it.
We can most probably later safely delete them in the chart transformation process.

**Comment by RedDog132nd** [comment]
I'd like to work on a better ground chart for Bodo. By better I mean more readable
I probably will make 3 more charts for zoomed in parking positions. I have in mind Kilo area, Golf Area and North Apron

What are the requirement for such a chart set?
Anything specific I should look for? 
Are parking positions per squadron/type of aircrfat set or are still changing or are to remain dynamic? 
Is the flow (direction) of movement in the K, G area set and defined? 
Do we need GPS cordinate for each shelter? 
Anything else I overlooked?

**Comment by Gabykhan** [comment]
Just in case it can be of any help for chart designers:
https://github.com/antoniolocandro/aeronautical_charting

**Comment by 132ndNeck** [comment]
> I'd like to work on a better ground chart for Bodo. By better I mean more readable I probably will make 3 more charts for zoomed in parking positions. I have in mind Kilo area, Golf Area and North Apron
> 
> What are the requirement for such a chart set? Anything specific I should look for? Are parking positions per squadron/type of aircrfat set or are still changing or are to remain dynamic? Is the flow (direction) of movement in the K, G area set and defined? Do we need GPS cordinate for each shelter? Anything else I overlooked?

That is great!
- Parking positions are mostly set and is per what is currently in the DCS mission editor.  There is not a specific area for each squadron and everyone is spread out around the airfield
- Flow have not been decided, and might not be needed? 
- GPS coordinates might be handy if going for a full INS alignmen, but might not be needed as people can get it from the DCS F10 map if needed. I can only imagine it is a lot of work to put it onto a chart and thus might not be worth the effort

**Comment by jkhoel** [comment]
Looking good @RedDog132nd ! The only thing I see right now is that the holding pattern will be inside of R11 I believe. And should the first "badge" (IAF) on the vertical chart read 10 instead perhaps?

**Comment by RedDog132nd** [comment]
TCN 07 Chart above update with revision number

The IAF is at 15 Nm as far as I can see, not 10Nm

The holding is not really necessary for us. I can delete it and instruct the MAP to turn left DCT BOO after 5Nm and make the holding at the IAF instead.

I usppose that it would be the same for the holdings at STOETT and ARDUX, although if inside one of ourr ange, we could also adapt the altitude restriction to ensure we are above the range airspace?

Let me know what the consensus would be and I can adapt the chart easy

**Comment by RedDog132nd** [comment]
Here's the same for the RWY25
Seems its a circle to land since the angle of the TCN approach is too different from the runway axis.
So higher MDA on this one:

![HPMA_TCN_25_ENBO_A5](https://github.com/user-attachments/assets/a1276fcc-0fa9-4526-8dc9-60ab8f54a4b3)

**Comment by RedDog132nd** [comment]
Is the ILS at Bodo operational?

**Comment by RedDog132nd** [comment]
Here's a draft update for the ENBO aerodrome chart

![Draft_ENBO_GROUND](https://github.com/user-attachments/assets/e49efdab-924c-4b78-85d5-1d63beca9d19)

List of change
- Higher resolution so hopefully better readibility
- new taxi layout corresponding to DCS
- revision added (this should be v0.7 - bottom right)
- Corrected ATIS freq

**Comment by jkhoel** [comment]
> 
> I suppose that it would be the same for the holdings at STOETT and ARDUX, although if inside one of ourr ange, we could also adapt the altitude restriction to ensure we are above the range airspace?
> 

Yea, I see the same thing on the second chart. This would be R13/R14. The altitude blocks for those ranges could be up in the FL250-FL300s if fixed wings are operating in there. I don't know what the optimal solution would be to be honest. In real life there would be controllers online to tell you if you could do your TACAN approach or not. But I think that if R14 is in use, then the last chart is a no-go, and if R11 is in use then the first chart is a no-go

**Comment by 132ndNeck** [comment]
> Looking good @RedDog132nd ! The only thing I see right now is that the holding pattern will be inside of R11 I believe. And should the first "badge" (IAF) on the vertical chart read 10 instead perhaps?

I do not see this as a major problem. Range 11 would likely not be in use when we use these charts.

As long as R11 is active then that holding can not be used, but range 11 is not that often in use. So I say we use our best judgement and SA on the event page when we prepare for the event.

**Comment by 132ndNeck** [comment]
> I probably need to erase also references to other tacans as I cannot find these in DCS
> OYA is probably ANDOYA but although we see a navaid 112.2 there, I cannot see a tacan channel
> BAR and MOS, no idea

Yes, I agree that we should erase everything not relevant to us in DCS. If it does not exist in DCS, it should not be there in the charts is my input.  I can probably put a TACAN wherever we want, using the portable TACAN in DCS, so I can place TACANs as required if needed

**Comment by 132ndNeck** [comment]
> List of change
> 
> * Higher resolution so hopefully better readibility
> * new taxi layout corresponding to DCS
> * revision added (this should be v0.7 - bottom right)
> * Corrected ATIS freq

Great work!
I suggest to remove the GND freq,and the second set of UHF freqs for TWR and APP.

**Comment by 132ndNeck** [comment]
> > I suppose that it would be the same for the holdings at STOETT and ARDUX, although if inside one of ourr ange, we could also adapt the altitude restriction to ensure we are above the range airspace?
> 
> Yea, I see the same thing on the second chart. This would be R13/R14. The altitude blocks for those ranges could be up in the FL250-FL300s if fixed wings are operating in there. I don't know what the optimal solution would be to be honest. In real life there would be controllers online to tell you if you could do your TACAN approach or not. But I think that if R14 is in use, then the last chart is a no-go, and if R11 is in use then the first chart is a no-go. Could set something like holding at FL250 and then have a certain block that the ranges can't operate in (like FL240-FL260) but then you are descending to land, so might still be wonky

I suggest that both charts can be used, and then we just make sure to check on the event info prior to the event if anyone is plannnig to use these ranges.

My main point is that I think we very rarely will come in situations where there is an actual conflict. And while in the real world the risk acceptance for that is 0, here in DCS, this beeing only a hobby and no one working full time to make sure everything works, I think it is better to accept that something might happen on a very rare occasion

**Comment by RedDog132nd** [comment]
> Yes, I agree that we should erase everything not relevant to us in DCS. If it does not exist in DCS, it should not be there in the charts is my input. I can probably put a TACAN wherever we want, using the portable TACAN in DCS, so I can place TACANs as required if needed

Done already. They have been erased from both TCN charts above

**Comment by RedDog132nd** [comment]
Here are the approach fixes and holding for the TCN 07 approach in Combat flight

![Fixes_TCN07_CF](https://github.com/user-attachments/assets/fe918cbb-67a8-4c99-8e82-e1b0115c0242)

The IAF holding is just a bit inside RNG 13 and RNG11
Ardux is in RNG 15
The MAP holding is no issue
The STOETT holding is smack above the targets of RNG 11

(but all these ranges are within the TMA ...)

I agree that when weather is sour, I hardly see anyone doing regular bombing range mission on R11 and we can coordinate that at mission planning or execution in the rare case it might happen.

I also think it might be interesting to add some fixes in CF:
IAF (need a name) - MAP (need a name) - ARDUX and STOETT comes to mind
But I'd wait on that so I can gather all of them in one go (other approach and SID) Maybe then add a category IFR  fixes?.

Testimony to your previous work is that ardux and stoett are exactly on the edge ot the TMA, perfectly where they are supposed to be !!

**Comment by 000Ready000** [comment]
> Is the ILS at Bodo operational?

About a month ago I tried ILS on RWY07 during a night flight. I was getting a localizer but maybe from the other runway heading. For the ILS I did not know the IAF and FAF. I tried Angels 9, which was too high.

This is what I checked:
ILS RWY 07
- ILS 110.3
- 84 degrees
TCN 45X BOO
117.55 BDO 122 (UNCLEAR)
WP FIX07 N 67°13.350 E 13°51.360

It would be great if there was an ILS approach chart. I searched but could not find any reference documents. Everything lead to dead links.

**Comment by RedDog132nd** [comment]
TCN 07 and ILS 07 above updated

V0.2: added CTR boundaries

**Comment by RedDog132nd** [comment]
Here's a SID chart
v0.2

![ENBO_HPMA_SID](https://github.com/user-attachments/assets/4d633869-8891-49e7-bbee-bd077a0b9bfc)

**Comment by RedDog132nd** [comment]
Reworking as well the HPMA VFR chart

Bigger, increased resolution, better readability, removed confusing stuff, will post it later

**Comment by RedDog132nd** [comment]
Last two: untested at this time:

ILS 25 ENBO :
![HPMA_ILS_RWY25_A5](https://github.com/user-attachments/assets/2c8acaf3-73ec-4c51-bbc1-9221c601f1e8)


TCN 25 ENBO
![HPMA_TCN_25_ENBO_A5](https://github.com/user-attachments/assets/b83678eb-20f9-4e7a-b3bb-71fc6bda38bb)

**Comment by RedDog132nd** [comment]
I'll need to modify the ILS 25 charts. the ILS worked on last training but the ILS in DCS is more offset and doesn't  match the real chart (274°). No big deal but it's confusing using the one above. So I'll have a new one created soon.

**Comment by RedDog132nd** [comment]
here's the final chart for ILS 25 with the correct offset
![HPMA_ILS_RWY25](https://github.com/user-attachments/assets/5a880aa0-57fd-48a8-83dd-ef691fbdc744)

**Comment by MatteoN74** [comment]
> Here's the update HPMA VFR chart
If the DEP instructions are to climb+maintain 2500', then maybe the NOTE box should have a line above AND below the 2500. I have always found that confusing that both 2500 and 2000 read "not above...", that may create conflict between DEP and ARR traffic.

**Comment by RedDog132nd** [comment]
It is my understanding that if you put a line under the 2500, you can't fly below 2500.
But within the same zone you take off so inevitably you will be under 2500 as you're climbing.
Same for arrival. You can't have a line under 2000 since you want to land and thus continue your descent.
so placing lines under 2500 (and under 2000) is not correct and counter-productive.

The line on top of 2500 prevents to exit the CTR vertically (and enter the TMA which is normally above it) the line above the 2000 for arrival gives a 500 feet cushion separation between the departing and arriving aircraft. eventhough the departing air do not HAVE to be AT 2500, they have to remain below 2500 and could be at 2000 if they are sloppy. 

It's not ideal but at the end it's VFR where ach pilot is responsible for its own deconfliction. That's the main rule

**Comment by Shadoga** [comment]
VFR CTR entry/exit procedures are never restricted to an exact altitude or have a line below an altitude to "cross at or above", because naturally the weather might dictate otherwise (to fly lower, as long as able to maintain VMC).
The 500 FT vertical separation between rotorwing, fixed wing and HPMA is just published for the optimum case of good weather (sufficiently high ceiling) allowing to deconflict traffic of different speed regimes vertically...

**Comment by 132nd-Entropy** [comment]
about the VAC for HPMA, what is the idea with teh VFR point Initials 07 ? 

or is it even a VFR point? Is there any sort of landmark that can be used to identify that point?

**Comment by 132nd-Entropy** [comment]
> Here's the update HPMA VFR chart
> 
> changes
> 
> * bigger size and better resolution
> * added version number
> * increases size for the hard to read data as reported above
> * changed ATIS frequency
> * simplified pattern
> * added vertical limits to CTR and TMA in a less confusing way
> * added MSA
> * added magvar (are the heading on the original charts true or magnetic?)
> * corrected elevation to match other charts
> 
> ![ENBO_VAC_HPMA_132nd_v06](https://github.com/user-attachments/assets/f9ab5323-b8ad-49cd-9008-e9aad8e1d43d)
> 
> As always comments welcome

the TACAN positions of the waypoints is missing in this version, I found those extremely useful to have

**Comment by RedDog132nd** [comment]
I can add them but is it really necessary? It's a visual chart. flown visually not with the tacan. Beside, INS coordinates are given top left so you can set you nav system to get a waypoint also.

**Comment by RedDog132nd** [comment]
> about the VAC for HPMA, what is the idea with teh VFR point Initials 07 ?
> or is it even a VFR point? Is there any sort of landmark that can be used to identify that point?

You have the runway in sight. You estimate or read the distance to it. You are initials. AFAIK, there's no visual cue for this one, it's all water. Your visual cue is the runway 5 Nm in front of you (AFAIK)

**Comment by SupremeBender** [comment]
Just a quick input on the VFR altitudes. I believe in real life they put arrivals above departures (ARR: 1500ft, DEP 1000ft), and since arrivals come in at 1500ft, they naturally fly the same altitude all the way from entry to initial. You will also never have altitude conflicts that way, since the climbers stop below the descenders.

This of course is all based on the MIL points (VOKKY, TWINS, CUTTY) that are all over water, so terrain is not considered.

**Comment by SupremeBender** [comment]
> Just came across this one. Looks like they don't do the overhead on 25.
> 
> <img width="694" height="952" alt="Image" src="https://github.com/user-attachments/assets/a5dd0343-bbe1-4ca5-84ea-4b06fc00eebd" />

I think the mountain to the northeast is in the way for a proper lineup, especially with a 4-ship, so they used to just do straight in single-ship landings on 25 and fly around to 07 if they wanted to do an OHB when coming in from the east

**Comment by argon39948** [comment]
> Here's the update HPMA VFR chart
> 
> changes
> 
> * bigger size and better resolution
> * added version number
> * increases size for the hard to read data as reported above
> * changed ATIS frequency
> * simplified pattern
> * added vertical limits to CTR and TMA in a less confusing way
> * added MSA
> * added magvar (are the heading on the original charts true or magnetic?)
> * corrected elevation to match other charts
> 
> ![ENBO_VAC_HPMA_132nd_v06](https://github.com/user-attachments/assets/f9ab5323-b8ad-49cd-9008-e9aad8e1d43d)
> 
> As always comments welcome

small typo in the coordinates of Waypoint Bjornoy Light LAT: N67°25.98'

**Comment by 132ndVarzat** [comment]
> > Just came across this one. Looks like they don't do the overhead on 25.
> > <img alt="Image" width="694" height="952" src="https://private-user-images.githubusercontent.com/225214134/475611608-a5dd0343-bbe1-4ca5-84ea-4b06fc00eebd.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjkxMDg4NDksIm5iZiI6MTc2OTEwODU0OSwicGF0aCI6Ii8yMjUyMTQxMzQvNDc1NjExNjA4LWE1ZGQwMzQzLWJiZTEtNGNhNS04NGVhLTRiMDZmYzAwZWViZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTIyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEyMlQxOTAyMjlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jNTU0NjY3M2RlNWU2MDBkZDczMGRkNDNhNTQyNTA3YmY3YjlkZDc4ZWIyNzc2YmI2OGNhZDk2Y2IwMGUyMzJhJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.kNHatwmN2b9ORGs8Kgw9E7yVeeHDPS0WD_Xbl7Y3oR4">
> 
> I think the mountain to the northeast is in the way for a proper lineup, especially with a 4-ship, so they used to just do straight in single-ship landings on 25 and fly around to 07 if they wanted to do an OHB when coming in from the east

Im super late to this but i think the reason for no overhead on 25 is probably noise abatment. The F-16s were always notorious with their sound, and the base leg would scream right overhead the roofs of the houses there. The mountain is not high enough to be an obstacle.
